src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormHeader.tag

```diff
@@ -0,0 +1,74 @@
+<%@ tag description="instructorFeedbacks - feedback session form header" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ tag import="teammates.common.util.Const" %>
+<%@ tag import="teammates.common.util.FieldValidator" %>
+<%@ tag import="teammates.logic.core.Emails.EmailType" %>
+
+<%@ attribute name="fsForm" type="teammates.ui.template.FeedbackSessionsForm" required="true"%>
+
+<c:if test="${fsForm.feedbackSessionTypeEditable}">
+    <div class="row">
+        <h4 class="label-control col-md-2 text-md">Create new </h4>
+        <div class="col-md-5">
+            <div class="col-md-10" title="Select a session type here."
+                data-toggle="tooltip" data-placement="top">
+                <select class="form-control"
+                    name="<%= Const.ParamsNames.FEEDBACK_SESSION_TYPE %>"
+                    id="<%= Const.ParamsNames.FEEDBACK_SESSION_TYPE %>">
+                    <c:forEach items="${fsForm.feedbackSessionTypeOptions}" var="option">
+                        <option <c:forEach items="${option.attributes}" var="attr"> ${attr.key}="${attr.value}"</c:forEach> >
+                            ${option.content}
+                        </option>
+                    </c:forEach>
+                </select>
+            </div>
+            <div class="col-md-1">
+                <h5>
+                    <a href="/instructorHelp.html#fbSetupSession" target="_blank">
+                        <span class="glyphicon glyphicon-info-sign"></span>
+                    </a>
+                </h5>
+            </div>
+        </div>
+        <h4 class="label-control col-md-1 text-md">Or: </h4>
+        <div class="col-md-3">
+            <a id="button_copy" class="btn btn-info" style="vertical-align:middle;">Loading...</a>
+        </div>
+    </div>
+</c:if>
+
+<c:if test="${fsForm.editFsButtonsVisible}">
+    <div class="row">
+        <div class="col-sm-12">
+            <span class="pull-right">
+                <a class="btn btn-primary btn-sm" id="fsEditLink"
+                    title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT %>"
+                    data-toggle="tooltip" data-placement="top"
+                    onclick="enableEditFS()">
+                    Edit
+                </a>
+                <button type="submit" id="fsSaveLink" style="display:none;" class="btn btn-primary btn-sm" onclick="return checkEditFeedbackSession(this.form);">
+                    Save Changes
```

Comment:
Is this tag still used?
I recall you setting "enableEditFS()" and "Save Changes" on the PageData side, are those irrelevant to this tag?